### ISSUE TITLE
Add a note about "async file logging" on README

### ISSUE THREAD

**Issue by daltonmatos** [B2W-BIT/aiologger#65]
When people come to aiologger, looking for a async logging solution (and they see that aiologger support loggin into files) they almost always think that the writes to the log files are fully async, but they aren't.

Since the Operating System (Linux, here) does not support fully async fie I/O, python (and aiologger) needs to use Threads to "simulate" this asynchronous writes.

We should state (and show with numbers) that aiologger is optimal for stdout/stderr logging and when you use this type logging you are achiving full async performance.

@diogommartins

**Comment by daltonmatos** [comment]
One possible draft of such text.

# A word about async, Python and files

Tldr; `aiologger` is only fully async when logging to stdout/stderr. If you log into files on disk you are not being fully async and will be using Threads.

`aiologger` was created when we realized that there were no async logging libs to use. At the time, Python's built-in logging infra-structure was fully sync (still is, 3.8 beta is out). That's why we created aiologger.

Despite everything (in Linux) being a file descriptor, a Network file descriptor and the stdout/stderr FDs are treated differently from files on disk FDs. This happens because there's no stable/usable async I/O interface published by the OS to be used by Python (or any other language). That's why **logging to files is NOT truly async**. `aiologger` implementation of file logging uses [aiofiles](https://github.com/Tinche/aiofiles), which uses a Thread Pool to write the data. Keep this in mind when using `aiologger` for file logging.

Other than that, we hope `aiologger` helps you write fully async apps. :tada: :tada:

**Comment by rubik** [comment]
@daltonmatos Very informative, thanks. I agree it's a good addition to the README.
